_data/simple-icons.json

```
@@ -13839,6 +13839,11 @@
             "hex": "002244",
             "source": "https://www.united.com"
         },
+        {
+            "title": "United Nations",
+            "hex": "009FDC",
```

```suggestion
            "hex": "009EDB",
```
Taken from guidelines